index.html

```diff
@@ -37,15 +37,15 @@
 
```

Comment:
Ay yo Patrick, how did you get permission to push to a branch?? I can't :(

Reply:
Ay yo Rebecca!! I had to fork it first to my own repo. Can @yabberyabber make this open so anyone can contribute? Is that even allowed on github?

Reply:
Ahh, gotcha, thanks!

Reply:
Can I make it open and still have gh-pages be a protected branch?

Reply:
Figured it out.  @mckinleyrebecca can you resolve conflict and I'll merge?

Reply:
Done!